### PR TITLE
Rel 5895: blackduck check compliance task 

### PR DESCRIPTION
**Pull request by sandeep-rawat-xebia**

### CODE REVIEW

File: src/main/resources/blackduck/task/helper/risk_profile.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                                             RiskProfile.__colored_value(self.__license_risk_low),
+                                             RiskProfile.__colored_value(self.__license_risk_none))
+        print "|{0}|{1}|{2}|{3}|{4}|".format('Operational Risk : ',
+                                             RiskProfile.__colored_value(self.__operational_risk_high),
+                                             RiskProfile.__colored_value(self.__operational_risk_medium),
+                                             RiskProfile.__colored_value(self.__operational_risk_low),
+                                             RiskProfile.__colored_value(self.__operational_risk_none))
+
+    @staticmethod
+    def __colored_value(risk_error):
```
Comment (pkumarxebia) [inline src/main/resources/blackduck/task/helper/risk_profile.py]: We may rename it to __get_display_text and move it to Risk class, Please fix formatting issue

File: src/main/resources/blackduck/task/helper/risk_profile.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        if self.__threshold is None or self.__value is None or self.__value <= self.__threshold:
+            return True
+        else:
+            return False
+
+    def get_value(self):
+        return self.__value
+
+
+class RiskProfile(object):
```
Comment (pkumarxebia) [inline src/main/resources/blackduck/task/helper/risk_profile.py]: RiskProfile may consist of risk_high, risk_medium, risk_low and risk_none, display_name with single print in pretty_print

File: src/main/resources/synthetic.xml [hunk 3]
```diff
@@ -31,4 +31,30 @@
         </property>
     </type>
 
+
+    <type type="blackduck.task" extends="xlrelease.PythonScript" virtual="true">
+        <property name="iconLocation" default="blackduck/blackduck.svg" hidden="true"/>
+        <property name="taskColor" hidden="true" default="#f84a06"/>
+        <property category="input" name="blackduckServer" referenced-type="blackduck.Server" kind="ci" label="Server"
+                  required="true" description="SONAR server id to connect to"/>
```
Comment (pkumarxebia) [inline src/main/resources/synthetic.xml]: SONAR in description!! Full stop is missing

File: src/main/resources/synthetic.xml [hunk 4]
```diff
@@ -31,4 +31,30 @@
...
+        <property category="input" name="projectName" required="true" label="Project Name" description="Name of project."/>
+    </type>
+
+
+    <type type="blackduck.checkCompliance" extends="blackduck.task" label="BlackDuck: Check Compliance">
+        <property name="configurationUri" hidden="true" default="include/BlackduckCheckComplianceTask/blackduck-check-compliance-task-view.html" />
+        <property name="scriptLocation" default="blackduck/task/check_compliance.py" hidden="true"/>
+        <property category="input" name="versionName" required="true" label="Version Name" description="Version of project."/>
```
Comment (pkumarxebia) [inline src/main/resources/synthetic.xml]: Can we have more details in description ending with full stop

File: src/main/resources/blackduck/task/helper/risk_profile.py (hunk 1 above)
Comment (sandeep-rawat-xebia) [reply src/main/resources/blackduck/task/helper/risk_profile.py]: Done

File: src/main/resources/blackduck/task/helper/risk_profile.py (hunk 2 above)
Comment (sandeep-rawat-xebia) [reply src/main/resources/blackduck/task/helper/risk_profile.py]: Done

File: src/main/resources/synthetic.xml (hunk 4 above)
Comment (sandeep-rawat-xebia) [reply src/main/resources/synthetic.xml]: Not possible

File: src/main/resources/synthetic.xml (hunk 3 above)
Comment (sandeep-rawat-xebia) [reply src/main/resources/synthetic.xml]: Done

**Pull request merged**